generators/server/templates/pom.xml.ejs

```diff
@@ -1019,13 +1016,11 @@
                             <artifactId>javassist</artifactId>
                             <version>${javassist.version}</version>
                         </dependency>
-                        <%_ if (databaseType === 'sql') { _%>
```

Comment:
We should not depend on liquibase for database not handled by liquibase, e.g. cassandra.
Why are you removing this condition?

Reply:
Ok, understood